<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1.1 (Python 3.13.0 on win32)" generated="20241114 19:30:09.574" rpa="false" schemaversion="4">
<suite id="s1" name="Tests" source="C:\Users\annik\OneDrive\Belgeler\kurssit\ohjelmistotuotanto\palautus\palautusrepo\osa3\login-robot\src\tests">
<suite id="s1-s1" name="Login" source="C:\Users\annik\OneDrive\Belgeler\kurssit\ohjelmistotuotanto\palautus\palautusrepo\osa3\login-robot\src\tests\login.robot">
<test id="s1-s1-t1" name="Login With Correct Credentials" line="9">
<kw name="Create User And Input Login Command" type="SETUP">
<kw name="Create User" library="AppLibrary">
<arg>kalle</arg>
<arg>kalle123</arg>
<msg timestamp="20241114 19:30:09.609" level="FAIL">AttributeError: 'UserService' object has no attribute 'find_all'</msg>
<status status="FAIL" starttime="20241114 19:30:09.609" endtime="20241114 19:30:09.611"/>
</kw>
<kw name="Input Login Command" library="resource">
<status status="NOT RUN" starttime="20241114 19:30:09.611" endtime="20241114 19:30:09.611"/>
</kw>
<status status="FAIL" starttime="20241114 19:30:09.609" endtime="20241114 19:30:09.612"/>
</kw>
<status status="FAIL" starttime="20241114 19:30:09.609" endtime="20241114 19:30:09.612">Setup failed:
AttributeError: 'UserService' object has no attribute 'find_all'</status>
</test>
<test id="s1-s1-t2" name="Login With Incorrect Password" line="13">
<kw name="Create User And Input Login Command" type="SETUP">
<kw name="Create User" library="AppLibrary">
<arg>kalle</arg>
<arg>kalle123</arg>
<msg timestamp="20241114 19:30:09.612" level="FAIL">AttributeError: 'UserService' object has no attribute 'find_all'</msg>
<status status="FAIL" starttime="20241114 19:30:09.612" endtime="20241114 19:30:09.613"/>
</kw>
<kw name="Input Login Command" library="resource">
<status status="NOT RUN" starttime="20241114 19:30:09.613" endtime="20241114 19:30:09.613"/>
</kw>
<status status="FAIL" starttime="20241114 19:30:09.612" endtime="20241114 19:30:09.613"/>
</kw>
<status status="FAIL" starttime="20241114 19:30:09.612" endtime="20241114 19:30:09.613">Setup failed:
AttributeError: 'UserService' object has no attribute 'find_all'</status>
</test>
<test id="s1-s1-t3" name="Login With Nonexistent Username" line="17">
<kw name="Create User And Input Login Command" type="SETUP">
<kw name="Create User" library="AppLibrary">
<arg>kalle</arg>
<arg>kalle123</arg>
<msg timestamp="20241114 19:30:09.614" level="FAIL">AttributeError: 'UserService' object has no attribute 'find_all'</msg>
<status status="FAIL" starttime="20241114 19:30:09.614" endtime="20241114 19:30:09.615"/>
</kw>
<kw name="Input Login Command" library="resource">
<status status="NOT RUN" starttime="20241114 19:30:09.615" endtime="20241114 19:30:09.615"/>
</kw>
<status status="FAIL" starttime="20241114 19:30:09.614" endtime="20241114 19:30:09.615"/>
</kw>
<status status="FAIL" starttime="20241114 19:30:09.614" endtime="20241114 19:30:09.615">Setup failed:
AttributeError: 'UserService' object has no attribute 'find_all'</status>
</test>
<status status="FAIL" starttime="20241114 19:30:09.596" endtime="20241114 19:30:09.615"/>
</suite>
<suite id="s1-s2" name="Register" source="C:\Users\annik\OneDrive\Belgeler\kurssit\ohjelmistotuotanto\palautus\palautusrepo\osa3\login-robot\src\tests\register.robot">
<test id="s1-s2-t1" name="Register With Valid Username And Password" line="10">
<kw name="Create User And Input Login Command" type="SETUP">
<kw name="Create User" library="AppLibrary">
<arg>kalle</arg>
<arg>kalle123</arg>
<msg timestamp="20241114 19:30:09.618" level="FAIL">AttributeError: 'UserService' object has no attribute 'find_all'</msg>
<status status="FAIL" starttime="20241114 19:30:09.618" endtime="20241114 19:30:09.620"/>
</kw>
<kw name="Input Login Command" library="resource">
<status status="NOT RUN" starttime="20241114 19:30:09.620" endtime="20241114 19:30:09.620"/>
</kw>
<status status="FAIL" starttime="20241114 19:30:09.618" endtime="20241114 19:30:09.620"/>
</kw>
<status status="FAIL" starttime="20241114 19:30:09.618" endtime="20241114 19:30:09.621">Setup failed:
AttributeError: 'UserService' object has no attribute 'find_all'</status>
</test>
<test id="s1-s2-t2" name="Register With Already Taken Username And Valid Password" line="13">
<kw name="Create User And Input Login Command" type="SETUP">
<kw name="Create User" library="AppLibrary">
<arg>kalle</arg>
<arg>kalle123</arg>
<msg timestamp="20241114 19:30:09.621" level="FAIL">AttributeError: 'UserService' object has no attribute 'find_all'</msg>
<status status="FAIL" starttime="20241114 19:30:09.621" endtime="20241114 19:30:09.622"/>
</kw>
<kw name="Input Login Command" library="resource">
<status status="NOT RUN" starttime="20241114 19:30:09.622" endtime="20241114 19:30:09.622"/>
</kw>
<status status="FAIL" starttime="20241114 19:30:09.621" endtime="20241114 19:30:09.622"/>
</kw>
<status status="FAIL" starttime="20241114 19:30:09.621" endtime="20241114 19:30:09.622">Setup failed:
AttributeError: 'UserService' object has no attribute 'find_all'</status>
</test>
<test id="s1-s2-t3" name="Register With Too Short Username And Valid Password" line="17">
<status status="FAIL" starttime="20241114 19:30:09.622" endtime="20241114 19:30:09.623">Test cannot be empty.</status>
</test>
<test id="s1-s2-t4" name="Register With Enough Long But Invalid Username And Valid Password" line="20">
<status status="FAIL" starttime="20241114 19:30:09.623" endtime="20241114 19:30:09.623">Test cannot be empty.</status>
</test>
<test id="s1-s2-t5" name="Register With Valid Username And Too Short Password" line="23">
<status status="FAIL" starttime="20241114 19:30:09.623" endtime="20241114 19:30:09.623">Test cannot be empty.</status>
</test>
<test id="s1-s2-t6" name="Register With Valid Username And Long Enough Password Containing Only Letters" line="26">
<status status="FAIL" starttime="20241114 19:30:09.623" endtime="20241114 19:30:09.623">Test cannot be empty.</status>
</test>
<status status="FAIL" starttime="20241114 19:30:09.616" endtime="20241114 19:30:09.623"/>
</suite>
<status status="FAIL" starttime="20241114 19:30:09.576" endtime="20241114 19:30:09.624"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="9" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="9" skip="0" id="s1" name="Tests">Tests</stat>
<stat pass="0" fail="3" skip="0" id="s1-s1" name="Login">Tests.Login</stat>
<stat pass="0" fail="6" skip="0" id="s1-s2" name="Register">Tests.Register</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20241114 19:30:09.565" level="ERROR">Error in file 'C:\Users\annik\OneDrive\Belgeler\kurssit\ohjelmistotuotanto\palautus\palautusrepo\osa3\login-robot\src\tests\login.robot' on line 4: Non-existing setting 'from entities.user import User'.</msg>
<msg timestamp="20241114 19:30:09.566" level="ERROR">Error in file 'C:\Users\annik\OneDrive\Belgeler\kurssit\ohjelmistotuotanto\palautus\palautusrepo\osa3\login-robot\src\tests\register.robot' on line 4: Non-existing setting 'from entities.user import User'.</msg>
<msg timestamp="20241114 19:30:09.567" level="ERROR">Error in file 'C:\Users\annik\OneDrive\Belgeler\kurssit\ohjelmistotuotanto\palautus\palautusrepo\osa3\login-robot\src\tests\register.robot' on line 7: Setting 'Test Setup' is allowed only once. Only the first value is used.</msg>
</errors>
</robot>
